### ISSUE TITLE
Limp Animation/Pose

### ISSUE THREAD

**Issue by JoshyCRB** [studio-minus/ppg-bugs-and-requests#1806]
**Feature description**
Makes people go limb. It's the same as when they are knocked out. Just a more usable version of the Flat Pose.

**Feature purpose**
It's easy to mess around with people while they are knocked out, but maybe you want it to be the same without them being knocked out. Now that I say that, it sounds kinda useless, but it would still be nice to have. Please consider it, thank you.

**Additional context**
The more features in a game, the better (or at least in most cases).

**Comment by DindinYT37** [comment]
It does sound pretty useless, yeah. Aren't you effectively asking for a knock out button in the context menu?

**Comment by DindinYT37** [comment]
> > It does sound pretty useless, yeah. Aren't you effectively asking for a knock out button in the context menu?
> 
> Yea, but they will wake up

So what's preventing you from knocking them out again? Also, if you wanted people that don't wake up again - kill them.

**Comment by SirAJ4895** [comment]
They will rot though

**Comment by DindinYT37** [comment]
> They will rot though

Yeah but what are you gonna use humans that won't wake up anyway?

**Comment by SirAJ4895** [comment]
Builds where you want a limp human, but without them rotting

**Comment by DindinYT37** [comment]
> Builds where you want a limp human, but without them rotting

Name an example.

**Comment by SirAJ4895** [comment]
If you have a hospital build and you want some people sleeping without them waking up or rotting. That's all I can think of.

**Comment by SirAJ4895** [comment]
Flat pose might  not work incase you want to drag them around
